arduino-hal/src/port/mega.rs

```diff
@@ -1,340 +1,343 @@
+        /// `PUB D53`
         ///
         /// * `SS`: SPI bus Slave Select
         /// * `PCINT0`: External Interrupt (Pin Change)
         pub d53: atmega_hal::port::PB0 = pb0,
-        /// `A0`
+        /// `PUB A0`
         ///
         /// * `ADC0`: A/D converter input 0
         pub a0: atmega_hal::port::PF0 = pf0,
-        /// `A1`
+        /// `PUB A1`
         ///
         /// * `ADC1`: A/D converter input 1
         pub a1: atmega_hal::port::PF1 = pf1,
-        /// `A2`
+        /// `PUB A2`
         ///
         /// * `ADC2`: A/D converter input 2
         pub a2: atmega_hal::port::PF2 = pf2,
-        /// `A3`
+        /// `PUB A3`
         ///
         /// * `ADC3`: A/D converter input 3
         pub a3: atmega_hal::port::PF3 = pf3,
-        /// `A4`
+        /// `PUB A4`
         ///
         /// * `ADC4`: A/D converter input 4
         /// * `TCK`: JTAG test clock
         pub a4: atmega_hal::port::PF4 = pf4,
-        /// `A5`
+        /// `PUB A5`
         ///
         /// * `ADC5`: A/D converter input 5
         /// * `TMS`: JTAG test mode select
         pub a5: atmega_hal::port::PF5 = pf5,
-        /// `A6`
+        /// `PUB A6`
         ///
         /// * `ADC6`: A/D converter input 6
         /// * `TDO`: JTAG test data output
         pub a6: atmega_hal::port::PF6 = pf6,
-        /// `A7`
+        /// `PUB A7`
         ///
         /// * `ADC7`: A/D converter input 7
         /// * `TDI`: JTAG test data input
         pub a7: atmega_hal::port::PF7 = pf7,
-        /// `A8`
+        /// `PUB A8`
         ///
         /// * `ADC8`: A/D converter input 8
         /// * `PCINT16`: External Interrupt (Pin Change)
         pub a8: atmega_hal::port::PK0 = pk0,
-        /// `A9`
+        /// `PUB A9`
         ///
         /// * `ADC9`: A/D converter input 9
         /// * `PCINT17`: External Interrupt (Pin Change)
         pub a9: atmega_hal::port::PK1 = pk1,
-        /// `A10`
+        /// `PUB A10`
         ///
         /// * `ADC10`: A/D converter input 10
         /// * `PCINT18`: External Interrupt (Pin Change)
         pub a10: atmega_hal::port::PK2 = pk2,
-        /// `A11`
+        /// `PUB A11`
         ///
         /// * `ADC11`: A/D converter input 11
         /// * `PCINT19`: External Interrupt (Pin Change)
         pub a11: atmega_hal::port::PK3 = pk3,
-        /// `A12`
+        /// `PUB A12`
         ///
         /// * `ADC12`: A/D converter input 12
         /// * `PCINT20`: External Interrupt (Pin Change)
         pub a12: atmega_hal::port::PK4 = pk4,
-        /// `A13`
+        /// `PUB A13`
         ///
         /// * `ADC13`: A/D converter input 13
         /// * `PCINT21`: External Interrupt (Pin Change)
         pub a13: atmega_hal::port::PK5 = pk5,
-        /// `A14`
+        /// `PUB A14`
         ///
         /// * `ADC14`: A/D converter input 14
         /// * `PCINT22`: External Interrupt (Pin Change)
         pub a14: atmega_hal::port::PK6 = pk6,
-        /// `A15`
+        /// `PUB A15`
```

Comment:
Looks like a lot of accidental changes in this file?

Reply:
Yes, this was the result of carefree regex substitution... 😆
I'm fixing it right away.